frontend/src/templates/Field/Paragraph/ParagraphField.tsx

```diff
@@ -24,9 +25,12 @@ export const ParagraphField = ({
       },
     },
   })
+
   return (
-    <ReactMarkdown components={mdComponents}>
-      {schema.description}
-    </ReactMarkdown>
+    <Box mt="1rem">
```

Comment:
any reason why you added a margin top to this? @justynoh

Reply:
shoot, no, no reason. should be deleted, i must have missed it from copy paste oops

Reply:
[7cbbfff](https://github.com/opengovsg/FormSG/pull/5105/commits/7cbbfff14b8b887817d5f4d8ae55d8b05023d9b3)